server/coordinator.go

```diff
@@ -234,6 +234,7 @@ func (c *coordinator) addOperator(op Operator) bool {
 		log.Infof("coordinator: add operator %+v with higher priority, remove operator: %+v", op, old)
 	}
 
+	c.histories.add(regionID, op)
```

Comment:
The _op_ is added to history when _removeOperator_ too. https://github.com/pingcap/pd/pull/637/files#diff-414579b44396bf49088b1219e107ea44R262